src/lintsampler/grid.py

```diff
@@ -379,15 +379,17 @@ def choose(self, u):
         
         Parameters
         ----------
-        u : 1D array of ints, shape (N,)
+        u : 1D array of floats, shape (N,)
```

Comment:
Does this have to be floats? I'm reasonably certain it can't be ints, though.

Reply:
Yes, good catch